helpers/mysql/lib/opentelemetry/helpers/mysql.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0module OpenTelemetry
+require 'opentelemetry-common'
+
+module OpenTelemetry
+  module Helpers
+    # This class contains helpers for MySQL instrumentation libraries, like mysql2 and
+    # trilogy. It is intended for use by instrumentation developers and not to
+    # be called directly by an application.
+    #
+    # To use this in your instrumentation, the `Instrumentation` class for
+    # your gem must contain configuration options for:
+    #  * `:span_name`
+    #  Example:
+    #  `option :span_name, default: :statement_type, validate: %I[statement_type db_name db_operation_and_name]`
+    #
+    # @api public
+    class MySQL
+      QUERY_NAMES = [
+        'begin',
+        'commit',
+        'create database',
+        'create table',
+        'delete',
+        'drop database',
+        'drop table',
+        'explain',
+        'insert',
+        'prepare statement',
```

Comment:
@xuan-cao-swi - 'prepare statement' is now included in the list of MySQL query names.

Reply:
This has been removed. @xuan-cao-swi let me know that the prepare method will need to be instrumented to make this work successfully. This work should be taken up in a separate PR. See: https://github.com/open-telemetry/opentelemetry-ruby-contrib/issues/670